packages/design-system/src/components/Table/TableCaption.jsx

```diff
@@ -8,11 +8,11 @@ export const TableCaption = ({
   _id,
   _scrollActive,
   _scrollableNotice,
-  ...others
+  ...tableCaptionProps
 }) => {
   const classes = classNames('ds-c-table__caption', className);
   return (
-    <caption className={classes} id={_id} {...others}>
+    <caption className={classes} id={_id} {...tableCaptionProps}>
       {children}
       {_scrollActive && _scrollableNotice}
     </caption>
```

Comment:
I don't think className needs to be inside the react prop docs: https://github.com/CMSgov/design-system/pull/796/files#diff-265b21c2ce32ee008f705e6c9081ce0fR30, as we pretty much want the user to treat `TableCaption`, `TableBody` and `TableHead` the same as the native HTML.

Could you add the @hide-prop tag to it?

Reply:
Will address this later